vercel/.env

```
@@ -0,0 +1,28 @@
+# Facebook Auth Configuration
+FACEBOOK_APP_ID=1138740481423987
+FACEBOOK_APP_SECRET=28df88bc6ea9ddb608e72efab17b92ff
+FACEBOOK_CALLBACK_URL=/auth/facebook/callback
+
+# Session Configuration
+SESSION_SECRET=g)ue(ss# %m4e &i@f y25o*u c*69an
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)